package.json

```diff
@@ -55,6 +55,8 @@
     "chokidar": "^1.7.0",
     "css-loader": "^0.28.7",
     "enzyme": "^3.2.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.1",
```

Comment:
Why do you need all of this ?

Reply:
Adpater is needed for enzyme itself while enzyme to JSON gives more readable snapshots. I've used them on other snapshots tests not in this push.